src/store/__test__.ts

```diff
@@ -0,0 +1,111 @@
+import { Operators } from '../config/operatorConfigs';
```

Comment:
this file was removed previously and was replaced by boring. Maybe a bad merge brought it back from the dead.